docs/modules/ROOT/pages/about/comparison.adoc

```diff
@@ -0,0 +1,37 @@
+= Comparison to Similar Tools
+
+== Rancher Fleet
+
+https://fleet.rancher.io[Fleet is GitOps at scale].
+
+The main goal of Fleet is to deploy applications to remote clusters, called the Fleet agents, whereas Project Syn focuses on generating configuration prepared for each cluster, depending on the properties of the destination cluster.
+
+Fleet implements it's own GitOps model, rather than reusing existing tools like Flux or Argo CD like Project Syn does.
+
+Git repositories are watched for changes by the central component called Fleet Manager.
+Deployment artifacts are prepared for deployment to the destination clusters by generating a Helm Chart, putting it into a CRD which is transfered to the destination cluster by the Fleet agent. On the destination cluster the Helm Chart is installed.
+In Project Syn the configuration is generated and stored in a Git repository per cluster.
+These generated objects are then applied as-is directly on the destination cluster which gets it by accessing the Git repository with Argo CD.
+
+Together with Kapitan and Vault Project Syn solves the issue of storing secrets in a secure way.
+Fleet does not provide any solution for that out-of-the box.
+
+Bootstrapping Fleet into a destination cluster is done using a bootstrap token which is very similar to Project Syn.
+Fleet doesn't need any API server and does everything via the Kubernetes API while Project Syn relies on a specialized API which stores it's data on the underlying Kubernetes cluster.
+
+== CAOS ORBOS
+
+https://github.com/caos/orbos[ORBOS - GitOps everything].
+
+The tooling is composed of three main components:
+
+* ORBITER: Orchestration tooling to spin up and manage Kubernetes clusters.
+* BOOM: Kubernetes Operator to install and managed system tooling on a Kubernetes clusters.
+* MISSION: Web UI to manage Kubernetes clusters, closed source.
+
+In contrary to Project Syn ORBOS focuses on the full lifecycle of Kubernetes clusters, whereas Project Syn doesn't provide means to provision and managed Kubernetes Clusters.
+Project Syn delegates this part to specialized tooling and focuses on managing the content of a Kubernetes cluster.
+
+The Kubernetes Operator BOOM installs and manages a hardcoded set of tools and doesn't allow deviation from that list. Project Syn instead provides fully flexible configuration management based on composable and reusable components.
+
+None of these three tools provides means of securely managing secrets like Project Syn with Kapitan and Vault does.
```

Comment:
Structurally, this line belongs to the section _ORBOS_. From its meaning, It belongs to the document as a whole.

```suggestion
----

None of these three tools provides means of securely managing secrets like Project Syn with Kapitan and Vault does.
```

Or add a conclusion section.

Reply:
Refined.